### DISCUSSION TITLE
[Feature Request] Create a style package with generic styles

### DISCUSSION THREAD

**Discussion by jorenbroekema** [ing-bank/lion#284] [General] [answered]
The premise of Lion is to be white-label, barebone, unstyled. However, there are cases where we use some "technical" css to make things work or make things accessible, or look at least somewhat functional. 

Example: https://github.com/ing-bank/lion/pull/281#discussion_r327112811
The button in general is a good example, since it also has a background color, a hover effect, very basic and minimal, but still.

Might be nice to create a `style` package and put in for example mixins/srOnlyMixin.js for visually hiding elements that still need to be caught by assistive technologies.

In general, it will reduce tech debt, be more DRY, and provide a bit more consistency for the minimum styling that we do apply to our components.

**Comment by erikkroes** [comment]
Hi,

We are closing this issue because we're changing the way we handle feature requests. Your issue will get the "votes needed" label, and people can add 👍 reactions to [vote and show interest](https://github.com/ing-bank/lion/issues?q=label%3A%22votes+needed%22+sort%3Areactions-%2B1-desc). It will be reopened when this feature gets picked up. 

This we way we clearly focus on bugs in our open issues. At the same time we can see what community interest there is for new features.

Thanks!

**Comment by jorenbroekema** [comment, marked as answer]
Will add this when we find a need. At this moment SR Only is only used in one or two places and that's the only functional styling so far. Not enough to create a package for it.
